dashboard/client/src/pages/overview/cards/RecentJobsCard.tsx

```diff
@@ -98,70 +56,65 @@ const useRecentJobListItemStyles = makeStyles((theme) =>
-      color: "#036DCF",
-    },
-    entrypoint: {
-      overflow: "hidden",
-      textOverflow: "ellipsis",
-      whiteSpace: "nowrap",
-      color: "#5F6469",
-    },
   }),
 );
 
-type RecentJobListItemProps = {
-  job: UnifiedJob;
+type RecentJobsCardProps = {
   className?: string;
 };
 
-const RecentJobListItem = ({ job, className }: RecentJobListItemProps) => {
-  const classes = useRecentJobListItemStyles();
+export const RecentJobsCard = ({ className }: RecentJobsCardProps) => {
+  const classes = useStyles();
+
+  const { jobList } = useJobList();
+
+  const sortedJobs = _.orderBy(jobList, ["startTime"], ["desc"]).slice(0, 6);
+
+  const sortedJobToRender = sortedJobs.map((job) => {
+    const icon = (() => {
+      switch (job.status) {
+        case "SUCCEEDED":
+          return (
+            <RiCheckboxCircleFill
+              className={classNames(classes.icon, classes.colorSuccess)}
+            />
+          );
+        case "FAILED":
+        case "STOPPED":
+          return (
+            <RiCloseCircleFill
+              className={classNames(classes.icon, classes.colorError)}
+            />
+          );
+        default:
+          return (
+            <RiLoader4Line
+              className={classNames(classes.icon, classes.iconRunning)}
+            />
+          );
+      }
+    })();
+
+    return {
+      title: job.job_id ?? job.submission_id,
+      subtitle: job.entrypoint,
+      link:
+        job.job_id !== null && job.job_id !== ""
+          ? `?/jobs/${job.job_id}`
+          : undefined,
+      className: className,
+      icon: icon,
+    };
+  });
 
-  const icon = (() => {
-    switch (job.status) {
-      case "SUCCEEDED":
-        return (
-          <RiCheckboxCircleFill
-            className={classNames(classes.icon, classes.colorSuccess)}
-          />
-        );
-      case "FAILED":
-      case "STOPPED":
-        return (
-          <RiCloseCircleFill
-            className={classNames(classes.icon, classes.colorError)}
-          />
-        );
-      default:
-        return (
-          <RiLoader4Line
-            className={classNames(classes.icon, classes.iconRunning)}
-          />
-        );
-    }
-  })();
   return (
-    <div className={className}>
-      <Link className={classes.root} to={`/jobs/${job.job_id}`}>
-        {icon}
-        <div className={classes.textContainer}>
-          <Typography className={classes.title} variant="body2">
-            {job.job_id ?? job.submission_id}
-          </Typography>
-          <Typography
-            className={classes.entrypoint}
-            title={job.entrypoint}
-            variant="caption"
-          >
-            {job.entrypoint}
-          </Typography>
-        </div>
-      </Link>
-    </div>
+    <CommonRecentCard
+      headerTitle={"Recent jobs"}
+      className={className}
+      items={sortedJobToRender}
+      itemEmptyTip={"No jobs yet..."}
+      footerText={"View all jobs"}
+      footerLink={"/jobs"}
```

Comment:
For string properties, do not surround with `{}`

example: `footerText="View all jobs"`